plugins/org.polarsys.capella.cybersecurity.ui/src/org/polarsys/capella/cybersecurity/ui/CybersecurityProjectWizard.java

```diff
@@ -56,4 +66,39 @@ public Session createAirdSession(List<IFile> semanticFiles, URI airdResourceURI,
 
     };
   }
+  
+  @Override
+  public boolean performFinish() {
+    boolean result = super.performFinish();
+    
+    EObject rootModel = ActivityExplorerManager.INSTANCE.getRootSemanticModel();

```

Comment:
can you avoid use of ActivityExplorer api in this method.
ActivityExplorer can be disabled through preferences and not being opened after project creation.

You can create configuration while project creation, in above method createSessionCreaationHelper, there is some method to extends, maybe update'InitialElementsCommand

It will not work though when you use capella and enable the viewpoint after project being created by using 
Windows/Show view / Viewpoint manager / Reference Cybersecurity.

@tguiu do you know how to add programatically some elements after a viewpoint is referenced through view Viewpoint Manager ?
Thanks

Reply:
I have saved the session and used it to retrieve the project.

Reply:
You can use a listener to monitor such an event.  see ViewpointManager.OverallListener2 class.